### PR TITLE
Delete the A/B test logic from membership front-end

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?
The Checkout A/B Test has finished. Flow B won the test (_Though not by a statistically significant amount - Roberto_). Therefore, we remove the logic of the A/B Test and when we call the identity project we do it with the members' id only.

## Changes
* `CheckoutFlowVariant.scala`, `paymentB.scala.html`: Delete the files
* `Fallbacks.scala`, `Config.scala`,  `Joiner.scala`, `signedInAs.scala.html`: Remove the A/B logic.
* `payment.scala.html`: Apply flow B changes to payment view.

## Related PRs

https://github.com/guardian/identity-frontend/pull/197 - should be merged after this.

### Before
**Old checkout page**
production-promo-code-a

### After

**New checkout page**
checkoutb

### CODE REVIEW

**Comment by rtyley** [comment]
You can check the statistical significance of a result with https://vwo.com/ab-split-test-significance-calculator/ - see also https://trello.com/c/CVPYemLI/238-apply-new-checkout-flow-to-membership-identity

**Comment by svillafe** [comment]
@rtyley Thank you! The images have been added.

File: frontend/app/views/joiner/form/payment.scala.html [hunk 1]
```diff
@@ -78,32 +81,34 @@ <h2 class="form-group__title">Address</h2>
...
+                            )
+                        </div>
                         @fragments.form.paymentOptions(plans)
                         @fragments.form.cardDetail(plans)
                     </div>
-
+                    @fragments.form.terms(countryGroup)
                     @fragments.form.errorMessageDisplay()
-                    @fragments.form.submit(PaidPlans(plans), countryGroup = countryGroup, dataCategory = Some("checkout page"), dataLabel = Some("payment a-annual"))
+                    @fragments.form.submit(PaidPlans(plans), membershipTermsAndConditions = false, countryGroup = countryGroup, dataCategory = Some("checkout page"), dataLabel = Some("payment b-annual"))
```
Comment (Ap0c) [inline frontend/app/views/joiner/form/payment.scala.html]: Do we still need to pass through `membershipTermsAndConditions`?

Comment (svillafe) [inline frontend/app/views/joiner/form/payment.scala.html]: It is been used in other files e.g. ***freeToPaid.scala.html***, ***addressWithWelcomePack.scala.html***, etc.

File: frontend/app/views/joiner/form/payment.scala.html [hunk 2]
```diff
@@ -37,32 +36,36 @@
...
+                        </div>
+
+                        <div class="form-section__name-detail">
+                        @fragments.form.nameDetail(idUser.privateFields.firstName, idUser.privateFields.secondName, heading = "")
+                        </div>
                         @fragments.form.addressDetail(
                             countriesWithCurrencies = countriesWithCurrencies,
                             heading = "Delivery address",
-                            note = "Once you've joined Guardian Members we'll send you a welcome pack in the post.",
+                            note = "We need your address to send you .",
```
Comment (svillafe) [inline frontend/app/views/joiner/form/payment.scala.html]: "... a welcome pack." I fixed the line. Thanks :) !

File: frontend/app/views/joiner/form/payment.scala.html (hunk 1 above)
Comment (svillafe) [inline frontend/app/views/joiner/form/payment.scala.html]: Yes, I modified this line and also refactor the js event system.

**Comment by svillafe** [comment]
It is working on **PROD**, the new checkout page is being shown.

**Pull request merged**